recipes/pyfinance/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pyfinance" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e4f723b5e6e399baf1938031def0f9f7de7afccf990d19a37fb74b8093a43c4b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.*
+    - pip
+    - setuptools
+    - pytest-runner
```

Comment:
Do you really need the ``pytest-runner`` to package this lib?

Reply:
I see, it is necessary because of that
https://github.com/bsolomon1124/pyfinance/blob/5416db9f1605e96ca2702994af0ceba83aae82e6/setup.py#L37

Reply:
yup, an earlier build failed because of it